Kudu.Core/Functions/KedaFunctionTriggerProvider.cs

```diff
@@ -222,6 +231,58 @@ internal static bool TryGetDurableKedaTrigger(string hostJsonText, out ScaleTrig
             return scaleTrigger != null;
         }
 
+        /// <summary>
+        /// Tries to add a scale trigger if the app is a workflow app.
+        /// </summary>
+        /// <param name="hostJsonText">The host.json text.</param>
+        /// <param name="appName">The app name.</param>
+        /// <param name="scaleTrigger">The scale trigger.</param>
+        /// <returns>true if a scale trigger was found</returns>
+        internal static bool TryGetWorkflowKedaTrigger(string hostJsonText, string appName, out ScaleTrigger scaleTrigger)
+        {
+            scaleTrigger = null;
+
+            // Check if an app is a workflow app
+            JObject hostJson = JObject.Parse(hostJsonText);
+            var extensionId = hostJson["extensionBundle"]?["id"]?.ToString();
+            if (extensionId == Constants.WorkflowExtensionBundle) // TODO(check app setting) APP_KIND
```

Comment:
We will be able to check if the app kind is "workflowApp" once we have this merged: https://github.com/microsoft/k4apps/pull/451 so this should change